test/libyul/EwasmTranslationTest.cpp

```diff
@@ -108,13 +108,13 @@ string EwasmTranslationTest::interpret()
 	state.maxExprNesting = 64;
 	try
 	{
-		Interpreter::run(state, WasmDialect{}, *m_object->code);
+		Interpreter::run(state, WasmDialect{}, *m_object->code, false);
```

Comment:
Memory tracing is enabled (last argument = `false`) for Ewasm->Yul tests. This may be changed to `true` if tests pass only without memory tracing.